README.md

```diff
@@ -372,85 +386,69 @@ _Note: Deployments and changes have a many to one relationship.  Table only cont
   </tr>
 </table>
 
-
+Where the key icon indicates that the ID is generated by the original system, such as GitHub.
 
 # Dashboard 
 
 The dashboard displays all four metrics with daily systems data, as well as a current snapshot of the last 90 days.  
 
-To understand the metrics and intent of the dashboard, please see the [2019 State of DevOps Report.](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf) 
+To understand the metrics and intent of the dashboard, see the [2019 State of DevOps Report](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf). 
 
 
-## Metrics Definitions
+## Metrics definitions
 
 **Deployment Frequency**
 
-
-
 *   The number of deployments per time period: daily, weekly, monthly, yearly. 
 
 **Lead Time for Changes**
 
-
-
-*   The median amount of time for a commit to be deployed into production
+*   The median amount of time for a commit to be deployed into production.
 
 **Time to Restore Services**
 
-
-
-*   For a failure, the median amount of time between the deployment which caused the failure, and the restoration.  The restoration is measured by closing an associated bug / incident report. 
+*   For a failure, the median amount of time between the deployment which caused the failure and the restoration.  The restoration is measured by closing an associated bug / incident report. 
```

Comment:
Is the only change here removing double spaces?

Reply:
yes, I believe so - I don't think that spacing matters, but I may have changed some as I was going through.